src/main/java/it/gov/pagopa/payment/exception/ValidationExceptionHandler.java

```diff
@@ -0,0 +1,54 @@
+package it.gov.pagopa.payment.exception;
+
+import it.gov.pagopa.payment.dto.ErrorDTO;
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.MissingRequestHeaderException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+@Slf4j
+@Order(Ordered.HIGHEST_PRECEDENCE)
+public class ValidationExceptionHandler {
+
+  @ExceptionHandler(MethodArgumentNotValidException.class)
+  @ResponseStatus(HttpStatus.BAD_REQUEST)
+  public ErrorDTO handleValidationExceptions(
+      MethodArgumentNotValidException ex, HttpServletRequest request) {
+
+    String message = ex.getBindingResult().getAllErrors().stream()
+        .map(error -> {
+          String fieldName = ((FieldError) error).getField();
+          String errorMessage = error.getDefaultMessage();
+          return String.format("[%s]: %s", fieldName, errorMessage);
+        }).collect(Collectors.joining("; "));
+
+    log.info("A InvalidRequestException occurred handling request {}: HttpStatus 400 - {}",
```

Comment:
it's better to be more precise with the exception occurred, in order to have already an hint when reading the log
```suggestion
    log.info("A MethodArgumentNotValidException occurred handling request {}: HttpStatus 400 - {}",
```